MTPot.py

```diff
@@ -119,6 +188,18 @@ def handleException(self, exc_type, exc_param, exc_tb):
         honey_logger.debug(traceback.format_exc())
         return True
 
+    #Catch tracestack error
+    def inputcooker(self):
+        try:
+            TelnetHandler.inputcooker(self)
```

Comment:
As far as I understand, this style can only be used in python3. Otherwise, we need to change the class declaration to this: 
`class MyTelnetHandler(TelnetHandler, object):`
(see [this thread](http://stackoverflow.com/questions/1713038/super-fails-with-error-typeerror-argument-1-must-be-type-not-classobj) for more information)